### PR TITLE
Improve ZNP constants and typings

### PR DESCRIPTION
**Pull request by r0b1n**

### CODE REVIEW

File: src/adapter/z-stack/constants/common.ts [hunk 1]
```diff
@@ -315,4 +196,89 @@ const COMMON: {
     }
 };
 
-export default COMMON;
\ No newline at end of file
+export enum ZnpCommandStatus {
+    SUCCESS = 0x00,
+    FAILURE = 0x01,
+    INVALID_PARAM = 0x02,
+    NV_ITEM_UNINIT = 0x09,
```
Comment (Koenkk) [inline src/adapter/z-stack/constants/common.ts:203]: Wouldn't `NV_ITEM_INITIALIZED` be a better name?

From the docs:
> 0x09 = Success, item created and initialized

File: src/adapter/z-stack/constants/common.ts [hunk 2]
```diff
@@ -315,4 +196,89 @@ const COMMON: {
     }
 };
 
-export default COMMON;
\ No newline at end of file
+export enum ZnpCommandStatus {
```
Comment (Koenkk) [inline src/adapter/z-stack/constants/common.ts:199]: Why was this moved out of `COMMON` (in my opinion it makes it inconsistent with the rest).

**Comment by Koenkk** [comment]
Thanks for the PR, this is a good improvement!

Added some questions/comments.

File: src/adapter/z-stack/constants/common.ts (hunk 1 above)
Comment (r0b1n) [reply src/adapter/z-stack/constants/common.ts]: Good point, indeed. I copy-pasted the name from `SWRA442` doc. Apparently it is a typo in there. Renamed.

File: src/adapter/z-stack/constants/common.ts (hunk 2 above)
Comment (r0b1n) [reply src/adapter/z-stack/constants/common.ts:159]: Agree. Using some regexp magic I made them enums and moved everything under the COMMON namespace in `index.ts`. Also did a bit of cleanup in files which use those constants to make code a bit less verbose.

**Comment by r0b1n** [comment]
@Koenkk I did one more iteration, should be better now.

**Comment by Koenkk** [comment]
Looks great! Big thanks 🎖️
